install-streamnzb.sh

```
@@ -0,0 +1,1115 @@
+        return 0
+    fi
+    return 1
+}
+
+# Get Cloudflare Zone ID for domain
+get_cloudflare_zone_id() {
+    local token="$1"
+    local domain="$2"
+    
+    # Extract root domain (last two parts)
+    local root_domain=$(echo "$domain" | awk -F. '{print $(NF-1)"."$NF}')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)